plugin-development/README.md

```diff
@@ -248,7 +248,7 @@ Following is a snippet of `README.md` showing the project skeleton generated wit
 
 For example:
 
-```text
+````text
```

Comment:
It looks good, but could you add a hidden comment about this usage of 4 backquotes?
I have a concern about someone unexpectedly break original intent.

```
<!--- four backquotes is used because block contains three backquotes for markup -->
```